main.tf

```diff
@@ -0,0 +1,152 @@
+## Dynamic template for the flow logs according to the service the servcie that user want.
+ provider "aws" {
+   access_key = ""
```

Comment:
why we need secret keys .. we are assuming user will use role .. non need for this code

Reply:
also please create a seperate file provioder.tf